CalibTracker/SiStripCommon/plugins/ShallowTree.cc

```diff
@@ -12,6 +11,7 @@ ShallowTree::ShallowTree(const edm::ParameterSet& iConfig) {
 
   //int compSettings= iConfig.getParameter<int>("CompressionSettings",-1);
   int compSettings = iConfig.getUntrackedParameter<int>("CompressionSettings", -1);
+  edm::Service<TFileService> fs_;
```

Comment:
Since this variable is now local, one could follow the rules and remove the trailing underscore.

(And by the way, the same could be done for the local variable `handle_` at [L180](https://github.com/cms-sw/cmssw/blob/f806bfeeea28339e9422b358e76ca845d5b9f6af/CalibTracker/SiStripCommon/plugins/ShallowTree.cc#L180))